src/Template/Sadrs/xml/e2b.ctp

```diff
@@ -396,10 +407,10 @@
             </drug>
             <?php  endforeach; ?>
             <summary>
-                <narrativeincludeclinical><?php echo $sadr['description_of_reaction']; ?></narrativeincludeclinical>
-                <reportercomment><?php echo h($sadr['lab_test_results']); ?></reportercomment>
+                <narrativeincludeclinical><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '', $sadr['description_of_reaction']); ?></narrativeincludeclinical>
+                <reportercomment><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '', $sadr['lab_test_results']); ?></reportercomment>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate